webpack/assets/javascripts/react_app/components/common/forms/CommonForm.js

```diff
@@ -16,8 +16,7 @@ const CommonForm = ({
   >
     <label className="col-md-2 control-label">
       {label}
-      {required && ' *'}
```

Comment:
I realize I am a bit late for the party, but why swapping it? This gives you:

![required-field](https://user-images.githubusercontent.com/2664090/56732055-7a6b1100-675c-11e9-9582-eebf52b66c0c.png)

Is that intentional?

Reply:
I would guess not, and the meaning was to remove the space, but @amirfefer could maybe provide more intel.
Mine proposal got it back though: (https://github.com/theforeman/foreman/pull/6677/files#diff-053d24bb53b0f9345ba1a18ccdd3eb23R99) so I would appreciate the discussion there as the follow up should be to move to that one anyway.
Well if we want the changes of mine :stuck_out_tongue_closed_eyes:

Reply:
ah yeah, that probably wasn't intentional